src/components/controls/PasswordInput/README.md

```diff
@@ -0,0 +1,39 @@
+## Password Input
```

Comment:
Prepare the file for landing, use `LANDING_BLOCK` `GITHUB_BLOCK` containers. See other readmes as a reference